pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml

```diff
@@ -5,5 +5,7 @@
   .sponsor-logo-image
     %img{src:"/images/fit-250/AFE-dark-logo.png"}
 
+-# Todo: Use a real AFE France url. Still waiting for Amazon France to give us that link.
+- afe_link = request.country == 'FR' ? 'https://www.amazonfutureengineer.com/faq' : 'https://www.amazonfutureengineer.com'
```

Comment:
@wjordan Could you please confirm that I can use `request.country` in Pegasus to detect user locations based on their IPs? We also use the following code in other places to achieve the same goal. Not sure what are the differences between them. 
```haml
- require 'geocoder'
- require 'country_codes'
- location = Geocoder.search(request.ip).try(:first)
- us_ip = location.nil? || ['US', 'RD'].include?(location.country_code.to_s.upcase)
```